prime_backup/mcdr/online_player_counter.py

```diff
@@ -89,71 +82,99 @@ def query_thread():
 				self.logger.exception('Queried players from minecraft_data_api error', e)
 				return None
 			if result is None:
-				self.logger.warning('Queried players from minecraft_data_api failed')
+				self.logger.warning("Queried players from minecraft_data_api failed")
 				return None
 
 			with self.data_lock:
 				self.data_is_correct = True
-				self.online_player_list = player_names.copy()
-				self.player_record_list = player_names.copy()
+				self.player_dict = {
+					name: PlayerRecord(
+						name=name,
+						online=True,
+						valid=self.__is_valid_player(name),
+					)
+					for name in player_names
+				}
 
 			if log_success:
 				self.logger.info('Successfully queried online players from minecraft_data_api: {}'.format(player_names))
 
 		thread = threading.Thread(target=query_thread, name=misc_utils.make_thread_name('player-query'), daemon=True)
 		thread.start()
 
-	def on_load(self, prev: Any):
+	def on_load(self, prev: any):
```

Comment:
same as https://github.com/TISUnion/PrimeBackup/pull/86/commits/ebe19ffaa54fbf66577a96d5fb02d1ae72ce3177#r2241738069. Just don't perform refactors unrelated to this PR, don't introduce too much noise